sapi/fpm/tests/cpuaffinity-fail.phpt

```diff
@@ -0,0 +1,43 @@
+--TEST--
+FPM: cpu affinity test
+--SKIPIF--
+<?php include "skipif.inc";
+
+if (!str_contains(PHP_OS, 'Linux')) {
+    die('skipped supported only on Linux');
```

Comment:
FreeBSD supports `cpuset_setaffinity`, is there any reason not to test this?

Reply:
Yeah why is this limited to Linux only if `cpuset_setaffinity` is already implemented?